lib/svgo/coa.js

```diff
@@ -156,6 +160,9 @@ async function action(args, opts, command) {
     config.recursive = opts.recursive;
   }
 
+  // --exclude
+  config.exclude = opts.exclude ? opts.exclude.map(pattern => RegExp(pattern)) : [];
```

Comment:
Don't you need to `new` the `RegExp`? (`new RegExp(pattern)`)

Reply:
and maybe break the line up to make it a bit more readable?

```suggestion
  config.exclude = opts.exclude
    ? opts.exclude.map(pattern => RegExp(pattern))
    : [];
```